lib/rake/javaextensiontask.rb

```diff
@@ -212,13 +212,26 @@ def java_classpath_arg(*args)
         rescue => e
         end
       end
+
       unless jruby_cpath
         libdir = RbConfig::CONFIG['libdir']
         if libdir.start_with? "classpath:"
           raise 'Cannot build with jruby-complete'
         end
```

Comment:
Do we still need this condition?

Reply:
Yep. While compiling with jruby-complete, the libdir is `uri:classloader:/META-INF/jruby.home/lib` which anyway doesn't contain `jruby.jar`. In this case, location of `jruby-complete.jar` is itself a classpath which is not set by default for Java 9 onwards as mentioned in the description of https://github.com/jruby/activerecord-jdbc-adapter/commit/995be701d58f58a34bc0d1448c9cd54f9fcd9e48.

Reply:
I see.